app/lib/theme/CustomDiffFields.tsx

```diff
@@ -98,12 +98,15 @@ const showNumberDiff = (
   oldData: number,
   newData: number,
   isMoney: boolean,
-  isPercentage: boolean
+  isPercentage: boolean,
+  numberOfDecimalPlaces: number = 0
 ) => (
   <>
     <span id={id && `${id}-diffOld`} className="diffOld">
       <NumberFormat
         thousandSeparator
+        fixedDecimalScale={true}
+        decimalScale={isMoney ? 2 : numberOfDecimalPlaces ?? 0}
```

Comment:
could use some parentheses, also this logic should be written once and stored in a variable

Reply:
My formatter took out the parentheses